include/seqan3/argument_parser/detail/format_parse.hpp

```diff
@@ -17,6 +17,7 @@
 #include <vector>
 
 #include <seqan3/argument_parser/detail/format_base.hpp>
+#include <seqan3/core/char_operations/predicate.hpp>
```

Comment:
But this is still necessary for the argument parser?

Reply:
the argument parser uses right now some functionality of `char_operations/predicate.hpp`, but it forgot to add the include, because it was indirectly included by `seqan3/std/charconv`, which the argument parser also includes.